ckanext/recombinant/controller.py

```diff
@@ -340,7 +375,7 @@ def preview_table(self, resource_name, owner_org, errors=None):
             'dataset': dataset,
             'resource': r,
             'organization': org,
-            'errors': errors,
+            'errors':errors
```

Comment:
keep the space after `:` and trailing commas are totally fine (even encouraged) in python.